### ISSUE TITLE
Lambda Function: Using the ZipFile attribute requires a runtime to be specified

### ISSUE THREAD

**Issue by ericzbeard** [aws-cloudformation/cfn-lint#2683]
### CloudFormation Lint Version

cfn-lint 0.72.2

### What operating system are you using?

Mac

### Describe the bug

Did not receive a warning from `cfn-lint` for this template.

Resource handler returned message: “Invalid request provided: Using the ZipFile attribute requires a runtime to be specified”
AWSTemplateFormatVersion: "2010-09-09"

```yaml
Description: Example AWS CloudFormation template snippet.

Resources:
  Test:
    Type: AWS::Lambda::Function
    Properties:
      Role: arn:aws:iam::755952356119:role/lambda-basic
      Code:
        ZipFile: >
          """Example."""
          import boto3
          TEST = 1
```

### Expected behavior

Expected an error message from cfn-lint to tell me that I forgot to specify `Runtime`.

### Reproduction template

```yaml
Description: Example AWS CloudFormation template snippet.

Resources:
  Test:
    Type: AWS::Lambda::Function
    Properties:
      Role: arn:aws:iam::755952356119:role/lambda-basic
      Code:
        ZipFile: >
          """Example."""
          import boto3
          TEST = 1
```
